esx_service/vmdk_ops.py

```diff
@@ -104,6 +104,10 @@
 MAX_SKIP_COUNT = 16       # max retries on VMCI Get Ops failures
 VMDK_ADAPTER_TYPE = 'busLogic'  # default adapter type
 
+#VMCI Version
+# !!! server side VMCI version, need to change according with client side version !!! 
+VMCI_VERSION = "12"
```

Comment:
It does not really matter. To avoid confusion, I suggest naming it PROTOCOL_VERSION and assigning a value of 2 . It has nothing to do with release number

Also -please change comment to so something like `Server side understand protocol version. If you are changing client/server protocol we use over VMCI, PLEASE DO NOT FORGET TO CHANGE IT FOR CLIENT in file <file name> !`